CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# etcd-wrapper maintainers
+*   @gardener/etcd-druid-maintainers
```

Comment:
can we consider renaming this to `etcd-maintainers` and as you suggested use the same across all etcd projects?

Reply:
As agreed we will do this together later for all etcd repositories.